{
  "uid" : "312573e6bf9c53ec",
  "name" : "Verify presence, visibility and attributes of the image in the section 1",
  "fullName" : "tests.start_unauthorized_page_test.TestStartUnauthorizedPageImage#test_su_04_01_verify_image_in_section1",
  "historyId" : "1aafb9d66326b6a4ae885f268ca195f1",
  "time" : {
    "start" : 1728781345611,
    "stop" : 1728781345611,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f1dcb2c02a <unknown>\n#1 0x55f1dc8125e0 <unknown>\n#2 0x55f1dc80a7eb <unknown>\n#3 0x55f1dc7faa09 <unknown>\n#4 0x55f1dc7fc744 <unknown>\n#5 0x55f1dc7facbd <unknown>\n#6 0x55f1dc7fa52f <unknown>\n#7 0x55f1dc7fa422 <unknown>\n#8 0x55f1dc7f8493 <unknown>\n#9 0x55f1dc7f8ada <unknown>\n#10 0x55f1dc814f41 <unknown>\n#11 0x55f1dc8a68c5 <unknown>\n#12 0x55f1dc886b22 <unknown>\n#13 0x55f1dc8a5d7d <unknown>\n#14 0x55f1dc8868c3 <unknown>\n#15 0x55f1dc8546b3 <unknown>\n#16 0x55f1dc85568e <unknown>\n#17 0x55f1dcaf6a2b <unknown>\n#18 0x55f1dcafa9b1 <unknown>\n#19 0x55f1dcae3225 <unknown>\n#20 0x55f1dcafb532 <unknown>\n#21 0x55f1dcac838f <unknown>\n#22 0x55f1dcb1af28 <unknown>\n#23 0x55f1dcb1b0f3 <unknown>\n#24 0x55f1dcb2ae7c <unknown>\n#25 0x7fc9f4e94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4eefc4c8ff88ce1baca9d4d2058aaf8a\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c0e1a690>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55f1dcb1b0f3 \\\\u003Cunknown>\\\\n#24 0x55f1dcb2ae7c \\\\u003Cunknown>\\\\n#25 0x7fc9f4e94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55f1dcb2c02a <unknown>\nE       #1 0x55f1dc8125e0 <unknown>\nE       #2 0x55f1dc80a7eb <unknown>\nE       #3 0x55f1dc7faa09 <unknown>\nE       #4 0x55f1dc7fc744 <unknown>\nE       #5 0x55f1dc7facbd <unknown>\nE       #6 0x55f1dc7fa52f <unknown>\nE       #7 0x55f1dc7fa422 <unknown>\nE       #8 0x55f1dc7f8493 <unknown>\nE       #9 0x55f1dc7f8ada <unknown>\nE       #10 0x55f1dc814f41 <unknown>\nE       #11 0x55f1dc8a68c5 <unknown>\nE       #12 0x55f1dc886b22 <unknown>\nE       #13 0x55f1dc8a5d7d <unknown>\nE       #14 0x55f1dc8868c3 <unknown>\nE       #15 0x55f1dc8546b3 <unknown>\nE       #16 0x55f1dc85568e <unknown>\nE       #17 0x55f1dcaf6a2b <unknown>\nE       #18 0x55f1dcafa9b1 <unknown>\nE       #19 0x55f1dcae3225 <unknown>\nE       #20 0x55f1dcafb532 <unknown>\nE       #21 0x55f1dcac838f <unknown>\nE       #22 0x55f1dcb1af28 <unknown>\nE       #23 0x55f1dcb1b0f3 <unknown>\nE       #24 0x55f1dcb2ae7c <unknown>\nE       #25 0x7fc9f4e94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781345612,
      "stop" : 1728781346382,
      "duration" : 770
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781346383,
      "stop" : 1728781346969,
      "duration" : 586
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f1dcb2c02a <unknown>\n#1 0x55f1dc8125e0 <unknown>\n#2 0x55f1dc80a7eb <unknown>\n#3 0x55f1dc7faa09 <unknown>\n#4 0x55f1dc7fc744 <unknown>\n#5 0x55f1dc7facbd <unknown>\n#6 0x55f1dc7fa52f <unknown>\n#7 0x55f1dc7fa422 <unknown>\n#8 0x55f1dc7f8493 <unknown>\n#9 0x55f1dc7f8ada <unknown>\n#10 0x55f1dc814f41 <unknown>\n#11 0x55f1dc8a68c5 <unknown>\n#12 0x55f1dc886b22 <unknown>\n#13 0x55f1dc8a5d7d <unknown>\n#14 0x55f1dc8868c3 <unknown>\n#15 0x55f1dc8546b3 <unknown>\n#16 0x55f1dc85568e <unknown>\n#17 0x55f1dcaf6a2b <unknown>\n#18 0x55f1dcafa9b1 <unknown>\n#19 0x55f1dcae3225 <unknown>\n#20 0x55f1dcafb532 <unknown>\n#21 0x55f1dcac838f <unknown>\n#22 0x55f1dcb1af28 <unknown>\n#23 0x55f1dcb1b0f3 <unknown>\n#24 0x55f1dcb2ae7c <unknown>\n#25 0x7fc9f4e94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781346383,
        "stop" : 1728781346968,
        "duration" : 585
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55f1dcb2c02a <unknown>\n#1 0x55f1dc8125e0 <unknown>\n#2 0x55f1dc80a7eb <unknown>\n#3 0x55f1dc7faa09 <unknown>\n#4 0x55f1dc7fc744 <unknown>\n#5 0x55f1dc7facbd <unknown>\n#6 0x55f1dc7fa52f <unknown>\n#7 0x55f1dc7fa422 <unknown>\n#8 0x55f1dc7f8493 <unknown>\n#9 0x55f1dc7f8ada <unknown>\n#10 0x55f1dc814f41 <unknown>\n#11 0x55f1dc8a68c5 <unknown>\n#12 0x55f1dc886b22 <unknown>\n#13 0x55f1dc8a5d7d <unknown>\n#14 0x55f1dc8868c3 <unknown>\n#15 0x55f1dc8546b3 <unknown>\n#16 0x55f1dc85568e <unknown>\n#17 0x55f1dcaf6a2b <unknown>\n#18 0x55f1dcafa9b1 <unknown>\n#19 0x55f1dcae3225 <unknown>\n#20 0x55f1dcafb532 <unknown>\n#21 0x55f1dcac838f <unknown>\n#22 0x55f1dcb1af28 <unknown>\n#23 0x55f1dcb1b0f3 <unknown>\n#24 0x55f1dcb2ae7c <unknown>\n#25 0x7fc9f4e94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"4eefc4c8ff88ce1baca9d4d2058aaf8a\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781347001,
      "stop" : 1728781347064,
      "duration" : 63
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Start Unauthorized Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "start_unauthorized_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestStartUnauthorizedPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.start_unauthorized_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 6,
        "skipped" : 0,
        "passed" : 385,
        "unknown" : 0,
        "total" : 391
      },
      "items" : [ {
        "uid" : "6cc810d638ad8209",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/6cc810d638ad8209",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55c03769702a <unknown>\n#1 0x55c03737d5e0 <unknown>\n#2 0x55c0373757eb <unknown>\n#3 0x55c037365a09 <unknown>\n#4 0x55c037367744 <unknown>\n#5 0x55c037365cbd <unknown>\n#6 0x55c03736552f <unknown>\n#7 0x55c037365422 <unknown>\n#8 0x55c037363493 <unknown>\n#9 0x55c037363ada <unknown>\n#10 0x55c03737ff41 <unknown>\n#11 0x55c0374118c5 <unknown>\n#12 0x55c0373f1b22 <unknown>\n#13 0x55c037410d7d <unknown>\n#14 0x55c0373f18c3 <unknown>\n#15 0x55c0373bf6b3 <unknown>\n#16 0x55c0373c068e <unknown>\n#17 0x55c037661a2b <unknown>\n#18 0x55c0376659b1 <unknown>\n#19 0x55c03764e225 <unknown>\n#20 0x55c037666532 <unknown>\n#21 0x55c03763338f <unknown>\n#22 0x55c037685f28 <unknown>\n#23 0x55c0376860f3 <unknown>\n#24 0x55c037695e7c <unknown>\n#25 0x7f2fb2294ac3 <unknown>",
        "time" : {
          "start" : 1728694671807,
          "stop" : 1728694671807,
          "duration" : 0
        }
      }, {
        "uid" : "9177aa616ae2becf",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/9177aa616ae2becf",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55d676ff502a <unknown>\n#1 0x55d676cdb5e0 <unknown>\n#2 0x55d676cd37eb <unknown>\n#3 0x55d676cc3a09 <unknown>\n#4 0x55d676cc5744 <unknown>\n#5 0x55d676cc3cbd <unknown>\n#6 0x55d676cc352f <unknown>\n#7 0x55d676cc3422 <unknown>\n#8 0x55d676cc1493 <unknown>\n#9 0x55d676cc1ada <unknown>\n#10 0x55d676cddf41 <unknown>\n#11 0x55d676d6f8c5 <unknown>\n#12 0x55d676d4fb22 <unknown>\n#13 0x55d676d6ed7d <unknown>\n#14 0x55d676d4f8c3 <unknown>\n#15 0x55d676d1d6b3 <unknown>\n#16 0x55d676d1e68e <unknown>\n#17 0x55d676fbfa2b <unknown>\n#18 0x55d676fc39b1 <unknown>\n#19 0x55d676fac225 <unknown>\n#20 0x55d676fc4532 <unknown>\n#21 0x55d676f9138f <unknown>\n#22 0x55d676fe3f28 <unknown>\n#23 0x55d676fe40f3 <unknown>\n#24 0x55d676ff3e7c <unknown>\n#25 0x7f5d19894ac3 <unknown>",
        "time" : {
          "start" : 1728608274070,
          "stop" : 1728608274070,
          "duration" : 0
        }
      }, {
        "uid" : "3d57f992de737da",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/3d57f992de737da",
        "status" : "passed",
        "time" : {
          "start" : 1728557445344,
          "stop" : 1728557445429,
          "duration" : 85
        }
      }, {
        "uid" : "dda7f92f2b63a2d7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/dda7f92f2b63a2d7",
        "status" : "passed",
        "time" : {
          "start" : 1728554308277,
          "stop" : 1728554308358,
          "duration" : 81
        }
      }, {
        "uid" : "5d459061eada3eaf",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/5d459061eada3eaf",
        "status" : "passed",
        "time" : {
          "start" : 1728522321612,
          "stop" : 1728522321699,
          "duration" : 87
        }
      }, {
        "uid" : "b7924955311a1501",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/b7924955311a1501",
        "status" : "passed",
        "time" : {
          "start" : 1728470701471,
          "stop" : 1728470701562,
          "duration" : 91
        }
      }, {
        "uid" : "bcd0a7c30f4dc05b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/bcd0a7c30f4dc05b",
        "status" : "passed",
        "time" : {
          "start" : 1728468132827,
          "stop" : 1728468132935,
          "duration" : 108
        }
      }, {
        "uid" : "8d95a9d232bd798d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/8d95a9d232bd798d",
        "status" : "passed",
        "time" : {
          "start" : 1728435948291,
          "stop" : 1728435948381,
          "duration" : 90
        }
      }, {
        "uid" : "75ec5db7ea7c6788",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/75ec5db7ea7c6788",
        "status" : "passed",
        "time" : {
          "start" : 1728378144619,
          "stop" : 1728378144710,
          "duration" : 91
        }
      }, {
        "uid" : "3f9ed8b799cbce55",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/3f9ed8b799cbce55",
        "status" : "passed",
        "time" : {
          "start" : 1728375354779,
          "stop" : 1728375354864,
          "duration" : 85
        }
      }, {
        "uid" : "defd57c7b2051d94",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/defd57c7b2051d94",
        "status" : "passed",
        "time" : {
          "start" : 1728375235777,
          "stop" : 1728375235863,
          "duration" : 86
        }
      }, {
        "uid" : "6aaf257f8019adac",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/6aaf257f8019adac",
        "status" : "passed",
        "time" : {
          "start" : 1728349324158,
          "stop" : 1728349324240,
          "duration" : 82
        }
      }, {
        "uid" : "f66ced4542c95d2f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/f66ced4542c95d2f",
        "status" : "passed",
        "time" : {
          "start" : 1728293073656,
          "stop" : 1728293073739,
          "duration" : 83
        }
      }, {
        "uid" : "d20af2fd23c7379e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/d20af2fd23c7379e",
        "status" : "passed",
        "time" : {
          "start" : 1728290125108,
          "stop" : 1728290125201,
          "duration" : 93
        }
      }, {
        "uid" : "e451699ab91fbfd9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/e451699ab91fbfd9",
        "status" : "passed",
        "time" : {
          "start" : 1728263770908,
          "stop" : 1728263770998,
          "duration" : 90
        }
      }, {
        "uid" : "4d5daa8b0f97aa94",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/4d5daa8b0f97aa94",
        "status" : "passed",
        "time" : {
          "start" : 1728226957445,
          "stop" : 1728226957533,
          "duration" : 88
        }
      }, {
        "uid" : "8e765906785ea45d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/8e765906785ea45d",
        "status" : "passed",
        "time" : {
          "start" : 1728222041316,
          "stop" : 1728222041400,
          "duration" : 84
        }
      }, {
        "uid" : "51f6d68e4dbf21db",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/51f6d68e4dbf21db",
        "status" : "passed",
        "time" : {
          "start" : 1728177138668,
          "stop" : 1728177138757,
          "duration" : 89
        }
      }, {
        "uid" : "5549403027e4f204",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/5549403027e4f204",
        "status" : "passed",
        "time" : {
          "start" : 1728117768821,
          "stop" : 1728117768902,
          "duration" : 81
        }
      }, {
        "uid" : "e045a494d5ba3e55",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/e045a494d5ba3e55",
        "status" : "passed",
        "time" : {
          "start" : 1728109144934,
          "stop" : 1728109145024,
          "duration" : 90
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "312573e6bf9c53ec.json",
  "parameterValues" : [ ]
}